### ISSUE TITLE
easey handlers seem broken with MM master

### ISSUE THREAD

**Issue by dhcole** [cutting-room-floor/easey-DEAD#11]

**Comment by mikemorris** [comment]
Getting this JavaScript error when trying to specify easey handlers:

```
modestmaps.js:2335
TypeError: 'undefined' is not a function (evaluating 'eventHandlers[j].init(this)')
```

Using
stamen/modestmaps-js@a2133e1efa0ebb900f41670fff27b8e398702e1e
mapbox/easey@70615c428c7f455c33f2dc7cf35bd945e87ea153
mapbox/wax@f606c82650ae8bb25a64b88b1fe73021d07a78ed

```
<script src="modestmaps-js/modestmaps.js" type="text/javascript"></script>
<script src="easey/src/easey.js" type="text/javascript"></script>
<script src="easey/src/easey.handlers.js" type="text/javascript"></script>
<script src="wax/dist/wax.mm.js" type="text/javascript"></script>
<script src="js/app.js" type="text/javascript"></script>
```

```
$(window).load(function() {
  var mm = com.modestmaps;
  var url = 'http://api.tiles.mapbox.com/v3/mapbox.mapbox-light.jsonp';

  wax.tilejson(url, function(tilejson) {
    var m = new mm.Map('map',
      new wax.mm.connector(tilejson),
      null, [
        new easey.DragHandler(),
        new easey.TouchHandler(),
        new easey.DoubleClickHandler(),
        new easey.MouseWheelHandler()
      ]);

    m.setCenterZoom({ lat: 38.8903694152832, lon: -77.0319595336914 }, 13);
    m.setZoomRange(13, 16);
  });
});
```

**Comment by tmcw** [comment]
Fixed in cf9d3fa3ce73a3f6451d1cdbab43a649756a9ac9
